### DISCUSSION TITLE
Go Pro 5 Unable to update to latest firmware

### DISCUSSION THREAD

**Discussion by carlo-p-tech** [gopro/labs#1176] [Q&A]
Hi,

I have a chicken and egg problem it seems.

My current and only GoPro Hero 5 original battery is not charging and is on constant 0%. This means I am unable to install the 2.7 update that allows rechargeable batteries. This means that I can't use a new battery to install the firmware update.

It seems that being directly powered via a cable is also not good enough. I also have tried a manual firmware update, but it still gives warning battery is too low.

I was wondering if there is any data transfer between the battery and the GoPro to know the battery %.

Would be a shame to bin the device when it still functions.

**Comment by dnewman-gpsw** [comment]
Hero 5 is not Labs enabled camera. I don't know anything about this.
